examples/django/README.md

```diff
@@ -0,0 +1,40 @@
+# Django Sample Application for Pybrake
+
+## About the application:
+
+The example application provides three GET endpoints:
+
+1. `/date` - returns server date and time. 
+2. `/locations` - returns list of available locations. 
+3. `/weather/{locationName}` - returns the weather details for the locations.
+
+## Steps to run the API:
+
+1. Install the dependencies for the application
+
+```bash
+pip3 install -r requirements.txt
```

Comment:
You need to add pybrake to `requirements.txt`. When I do `pip3 install` and then `python manage.py runserver 3000`, I get:

```
ModuleNotFoundError: No module named 'pybrake'
```